src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
I disagree, we should take a smaller value for other reasons such as:
* not forcing users to have a decent machine enough (because that's also one of the issues)
* we would find problems related to having a small tick sooner.